modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating clusters to a multi-arch/heterogenous payload using the CLI
```

Comment:
Using "clusters" plural seems inconsitent with other docs like https://docs.openshift.com/container-platform/4.11/updating/updating-cluster-cli.html - they all seem to use singular